README.md

```diff
@@ -27,7 +27,7 @@ There should be no breaking changes related to this transfer. Most of the extern
 Here is what is mostly affected by the transfer:
 
 - All the stars were transferred
-- Search in the code will not work **unless this repository has less stars than the parent** (your help is needed!)
```

Comment:
I can't verify if either wording is correct, because I'm not a native English speaker. From my point of view, the current wording is correct. I read it as follows: "Search in the code will not work *as long as* this repository has less stars than the parent". Let's just replace "unless" with "as long as"?

Reply:
That makes more sense. "Unless" in this case is whats throwing everything off